middleware.js

```diff
@@ -4,8 +4,7 @@ import App from './app/app'
 
 module.exports = (options) =>
   (req, res, next) => {
-    const endpointRegExp = new RegExp(options.middlewarePath, 'i')
-    if (endpointRegExp.test(req.url)) {
+    if (req.url.match(new RegExp(options.middlewarePath, 'i'))) {
```

Comment:
I think we could want to move the instantiation of the RegExp outside in a closure  so that it won't be done on every  req